### PR TITLE
Fix logic in actions

### PR DESCRIPTION
**Pull request by D4nte**
I have spotted some inconsistencies in the matching arms.

See commit messages for more details.

### CODE REVIEW

File: cnd/src/http_api/halight_herc20/alice.rs [hunk 1]
```diff
@@ -121,7 +121,7 @@ impl FundAction for AliceSwap<asset::Bitcoin, asset::Erc20, halight::Finalized,
                 },
                 beta_finalized:
                     herc20::Finalized {
-                        state: herc20::State::Funded { .. },
+                        state: herc20::State::None,
```
Comment (luckysori) [inline cnd/src/http_api/halight_herc20/alice.rs:124]: :exclamation: How did the e2e tests ever pass?

Comment (D4nte) [reply cnd/src/http_api/halight_herc20/alice.rs:124]: > Could we somehow assert the funding on the Lightning side for Alice in this scenario?

What do you mean? I have corrected that already in 2b3183144ed3a24f29e1232b422026e5813c6810 and 016b1dc4c8d371553ea1bf27ad7312ea4c7b2f2a.

**Comment by D4nte** [comment]
> Could we somehow assert the funding on the Lightning side for Alice in this scenario?

What do you mean? I have corrected that already in 2b3183144ed3a24f29e1232b422026e5813c6810 and 016b1dc4c8d371553ea1bf27ad7312ea4c7b2f2a.

**Comment by da-kami** [comment]
> > Could we somehow assert the funding on the Lightning side for Alice in this scenario?
> 
> What do you mean? I have corrected that already in [2b31831](https://github.com/comit-network/comit-rs/commit/2b3183144ed3a24f29e1232b422026e5813c6810) and [016b1dc](https://github.com/comit-network/comit-rs/commit/016b1dc4c8d371553ea1bf27ad7312ea4c7b2f2a).

I meant as assertion in the e2e test. The test should have failed, but it did not.

**Comment by D4nte** [comment]
> I meant as assertion in the e2e test. The test should have failed, but it did not. You fixed the code, I wonder if we can also be precise in the test.

Thanks, I guess my brain was mush at this stage of the day. I actually already improved the lightning payment so we should be able to await. Will try.

**Pull request merged**